### DISCUSSION TITLE
LinkIt ONE | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#639] [Q&A]
# LinkIt ONE | Seeed Studio Wiki

LinkIt ONE

https://wiki.seeedstudio.com/LinkIt_ONE/

**Comment by Juanmorales177809** [comment]
The links to download the sdk do not work and the github repository does not contain the SDK in the latest version, I could only find version 1.0.05 but it only works with arduino 1.5.7 Beta and I need it to work with arduino ide 1.6.2, Could you please share the SDK v 1.0.09 or higher
